exercicios/exercicio5.js

```diff
@@ -1 +1,15 @@
+// 5. Some uma lista de notas de escola
+
 const notas = [5, 4, 3, 0, 10]
+let soma = 0
+
+
+for(let i = 0; i < notas.length; i++){
```

Comment:
Legal sua solução com `for`! O que acha de tentar agora usando o [reduce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce)?